tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -1,5 +1,6 @@
 Unicode true
 ManifestDPIAware true
+ManifestDPIAwareness PerMonitorV2
```

Comment:
could you link to the documentation of this option? I can't seem to find a reliable source for it.

Reply:
This option is undocumented on their website, but is in the Docs folder of the source, I would probably prefer add in a link this PR instead, what do you think?

> https://github.com/kichik/nsis/blob/5fc0b87b819a9eec006df4967d08e522ddd651c9/Docs/src/attributes.but#L286-L300

Reply:
could add both links, no problem